www/templates/account/signup/step-2.php

```diff
@@ -46,28 +46,32 @@
 </section>
 <aside>
     <h3>Selected Plan</h3>
-    <div class="plan-name"><?= $is_plan_free ? "STARTER" : "PRO"; ?></div>
+    <div class="plan-name">
+        <?= $is_plan_free ? "STARTER" : '<div class="heading wpt-pro-logo"> <span class="visually-hidden">WebPageTest <em class="new-banner">Pro</em></span></div>' ?>
+    </div>
     <div class="plan-details">
         <table>
-            <thead>
-                <?php if (!$is_plan_free) : ?>
-                    <th>Runs per month</th>
-                <?php endif; ?>
-                <th>Price</th>
-            </thead>
-            <tbody>
-                <tr>
-                    <?php if ($is_plan_free) : ?>
-                        <td>Free</td>
+            <?php if( !$is_plan_free ) : ?>
+            <tr>
+                <th>Pay Plan:</th>
```

Comment:
This whole sidebar should be an include, so we don't repeat it OOOH NOOO I AM DRYing